resources/assets/components/pages/AccountLandingPage/AccountLandingPage.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import gql from 'graphql-tag';
+import { css } from '@emotion/core';
+
+import Query from '../../Query';
+import letsDoThis from './lets-do-this.png';
+import AccountIcon from './icons/AccountIcon';
+import RewardsIcon from './icons/RewardsIcon';
+import CampaignsIcon from './icons/CampaignsIcon';
+import { getUserId } from '../../../helpers/auth';
+import InterestsIcon from './icons/InterestsIcon';
+import { tailwind } from '../../../helpers/display';
+import ReferAFriendIcon from './icons/ReferAFriendIcon';
+import SubscriptionsIcon from './icons/SubscriptionsIcon';
+import SiteFooter from '../../utilities/SiteFooter/SiteFooter';
+import VolunteerCreditsIcon from './icons/VolunteerCreditsIcon';
+import SiteNavigation from '../../SiteNavigation/SiteNavigation';
+
+const accountLinks = [
+  {
+    title: 'My Campaigns',
+    slug: 'campaigns',
+    icon: CampaignsIcon,
+  },
+  {
+    title: 'My Rewards',
+    slug: 'rewards',
+    icon: RewardsIcon,
+  },
+  {
+    title: 'Volunteer Credits',
+    slug: 'credits',
+    icon: VolunteerCreditsIcon,
+  },
+  {
+    title: 'Refer A Friend',
+    subTitle: 'Enter to win a $10 gift card.',
+    slug: 'refer-friends',
+    icon: ReferAFriendIcon,
+  },
+  {
+    title: 'My Interests',
+    slug: 'interests',
+    icon: InterestsIcon,
+  },
+  {
+    title: 'My Subscriptions',
+    slug: 'subscriptions',
+    icon: SubscriptionsIcon,
+  },
+  {
+    title: 'My Account',
+    slug: 'profile',
+    icon: AccountIcon,
+  },
+];
+
+const USERNAME_QUERY = gql`
+  query UsernameQuery($userId: String!) {
```

Comment:
This is a nit but looks like we generally separate each word with a `_` in query names, so like:
```suggestion
const USER_NAME_QUERY = gql`
  query UserNameQuery($userId: String!) {
```

Reply:
To be fair, `username` is also a thing!